lib_nbgl/src/nbgl_use_case.c

```diff
@@ -152,9 +169,12 @@ static KeypadContext_t keypadContext;
 
 // contexts for generic navigation
 static GenericContext_t genericContext;
-static nbgl_content_t   localContentsList[2];
+static nbgl_content_t   localContentsList[3];
```

Comment:
Could you comment why this value `3`? Could it be a `define`, showing explicitely what it is?